mpdev/cmd/applycmd.go

```diff
@@ -42,11 +42,12 @@ func GetApplyCommand() *cobra.Command {
 	return cmd
 }
 
-type Command struct {
+type command struct {
 	Filenames []string
 }
 
-func (c *Command) RunE(_ *cobra.Command, _ []string) error {
+// Executes the `apply` command
```

Comment:
go doc requires the name of the function (or class, variable, etc) to be the first word